src/domain/Game.java

```diff
@@ -19,7 +19,9 @@
 	 */
 	private List<Player> activePlayers;
 	private int currentPlayerIndex = 0;
-
+	private List<Player> amountActivePlayers = activePlayers;

```

Comment:
Dit attribuut is niet nodig, om het aantal actieve spelers te krijgen kan je `activePlayers.size()` gebruiken